test/kubernetes_test.go

```diff
@@ -479,8 +840,63 @@ func TestKubernetesIntegrationFallthrough(t *testing.T) {
 	drop 0
     }
 `
-	cases := append(dnsTestCases, dnsTestCasesFallthrough...)
-	doIntegrationTests(t, corefile, cases)
+	doIntegrationTests(t, corefile, dnsTestCasesFallthrough)
+}
+
+func getDynamicEndpoints(Qname, namespace, name string) []dns.RR {
+	rr := []dns.RR{}
+
+	result := strings.Split(getCmdOutput(name, namespace), " ")
+	len_result := len(result)
+
+	if len_result > 1 {
+		for i := 0; i < len_result; i++ {
+			rr = append(rr, test.A(Qname+"    303    IN      A   "+result[i]))
+		}
+	} else {
+		rr = append(rr, test.A(Qname+"    303    IN      A   "+result[0]))
+	}
+
+	return rr
+}
+
+func getSRVrecords(Qname, Qtype, namespace, name string) []dns.RR {
+	rr := []dns.RR{}
+
+	result := strings.Split(getCmdOutput(name, namespace), " ")
+	len_result := len(result)
+
+	if len_result > 1 {
```

Comment:
Revert the flow here:
`len_result == 1 {
 // ip stuff
return
}
// srv stuff